packages/platforms/accton/x86-64/as5915-18x/onlp/builds/x86_64_accton_as5915_18x/module/src/sysi.c

```diff
@@ -150,3 +150,217 @@ int onlp_sysi_platform_manage_leds(void)
 
     return onlp_ledi_mode_set(ONLP_LED_ID_CREATE(LED_FAN), ONLP_LED_MODE_GREEN);
 }
+
+enum fan_duty {
+    FAN_DUTY_MAX = 100,    /* high speed */
+    FAN_DUTY_MID = 60,     /* middle speed */
+    FAN_DUTY_NOR = 30,     /* normal speed */
+    FAN_DUTY_LOW_1_3 = 0,  /* low speed for fan1 and fan3*/
+    FAN_DUTY_LOW_2_4 = 40  /* low speed for fan2 and fan4 */
```

Comment:
As 30 is normal speed, 40 doesn’t seem to be low?

Reply:
This is provided from our H/W team.
Due to Fan1/3 are disabled at low temperature, Fan2/4 need to set as 40 to avoid the failure of enabling Fan1/3 when temperature goes higher.